src/spectrecoff/Output.fs

```diff
@@ -97,29 +97,29 @@ let NL = NewLine
 let rec toMarkedUpString (payload: OutputPayload) =
     match payload with
     | Calm content -> content |> calm
-    | Pumped content -> content |> pumped 
+    | Pumped content -> content |> pumped
     | Edgy content -> content |> edgy
-    | Vanilla content -> content 
+    | Vanilla content -> content
     | MarkupCS (color, style, content) -> content |> markupString (Some color) (Some style)
     | MarkupC (color, content) -> content |> markupString (Some color) None
     | MarkupS (style, content) -> content |> markupString None (Some style)
     | Link link -> link |> markupWithColorAndStyle linkColor "link"
     | LinkWithLabel (label, link) -> label |> markupWithColorAndStyle linkColor $"link={link}"
     | Emoji emoji -> emoji |> padEmoji
-    | NewLine -> ""
-    | Collection items -> 
-        items 
+    | NewLine -> Environment.NewLine
```

Comment:
This change is not correct, imo., because I think it leads to _two new lines_. Please check and compare to make sure. If you look at your screenshot you can already see that the paragraphs are separated by two blank lines.

Reply:
![image](https://user-images.githubusercontent.com/17160319/207112530-471476ed-0a54-4bc9-9b21-8c744d32c068.png)

Reply:
@EluciusFTW, I am trouble finding the reason for this behavior. If you don't mind and there is no other issues here, I would like to close this PR and look at it in isolation.

Reply:
Shall we maybe take your solution in the intermidiary, 2 new lines, instead of none? I'd be fine closing it in that state as well.

Reply:
Done